internal/wasm/module_instance.go

```diff
@@ -13,6 +13,13 @@ import (
 // FailIfClosed returns a sys.ExitError if CloseWithExitCode was called.
 func (m *ModuleInstance) FailIfClosed() (err error) {
 	if closed := atomic.LoadUint64(&m.Closed); closed != 0 {
+		switch closed & 0xff {
```

Comment:
Is it worth declaring a const for the magic 0xff here? (e.g. `exitCodeFlagMask`)